### PR TITLE
Pretty print

### PR DESCRIPTION
**Pull request by atjn**
Adds a pretty printer, A.K.A a code formatter.

Also adds a bunch of fixes to issues discovered while implementing the pretty printer :)

### CODE REVIEW

File: src/main/java/dk/aau/p4/abaaja/Lib/Visitors/PrettyPrintVisitor.java [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+    public void visit(NumExpNode node){
+        // The `replaceAll` removes unnecessary trailing zeroes from output
+        print(node.get_result().toString().replaceAll("\\.?0*(?=$)", ""));
+    }
+    public void visit(StringExpNode node){
+        print(node.get_result());
+    }
+
+    /**
+     * Useless crap
```
Comment (BaBrixx) [inline src/main/java/dk/aau/p4/abaaja/Lib/Visitors/PrettyPrintVisitor.java:285]: Should this comment be changed? ❤️

Comment (atjn) [reply src/main/java/dk/aau/p4/abaaja/Lib/Visitors/PrettyPrintVisitor.java:285]: Sure, but it is true. You want it to be less spicy?

Comment (BaBrixx) [reply src/main/java/dk/aau/p4/abaaja/Lib/Visitors/PrettyPrintVisitor.java:285]: Yes please

Comment (atjn) [reply src/main/java/dk/aau/p4/abaaja/Lib/Visitors/PrettyPrintVisitor.java]: Fixed in [c840cc6](https://github.com/AlbertHald/Minecraft-Turtle-Language/pull/17/commits/c840cc612bfa0cb4cca281e6b66944e323ad743c)

**Comment by atjn** [comment]
TY <3

**Pull request merged**